pandas/core/generic.py

```diff
@@ -7576,11 +7523,10 @@ def _add_numeric_operations(cls):
         cls.any = _make_logical_function(
             cls, 'any', name, name2, axis_descr,
             'Return whether any element is True over requested axis',
-            nanops.nanany)
+            nanops.nanany, '', '')
```

Comment:
Maybe remove these so they don't conflict with the other PR.

Reply:
@TomAugspurger If I do this an error 
```python ../scripts/validate_docstrings.py pandas.DataFrame.all
Traceback (most recent call last):
  File "../scripts/validate_docstrings.py", line 37, in <module>
    import pandas
  File "/Users/marconlaforet/pandas-dev/pandas/__init__.py", line 42, in <module>
    from pandas.core.api import *
  File "/Users/marconlaforet/pandas-dev/pandas/core/api.py", line 10, in <module>
    from pandas.core.groupby import Grouper
  File "/Users/marconlaforet/pandas-dev/pandas/core/groupby.py", line 48, in <module>
    from pandas.core.frame import DataFrame
  File "/Users/marconlaforet/pandas-dev/pandas/core/frame.py", line 75, in <module>
    from pandas.core.series import Series
  File "/Users/marconlaforet/pandas-dev/pandas/core/series.py", line 3202, in <module>
    Series._add_numeric_operations()
  File "/Users/marconlaforet/pandas-dev/pandas/core/generic.py", line 7530, in _add_numeric_operations
    nanops.nanany)
TypeError: _make_logical_function() missing 2 required positional arguments: 'examples' and 'see_also'```

Reply:
Yes, you can keep this. Just depending on which one of the two PRs is merged first, you might need to solve the conflict on the other. But that should be no problem